problems/loops.js

```diff
@@ -4,31 +4,59 @@
  * @param {number}
  * @returns {number[]}
  */
-function oneTillDoneWhileLoop() {}
+function oneTillDoneWhileLoop(number) {
+  let newNum = [];
```

Comment:
variable name awareness. Arrays should be plural. or res / result / output / arr